doc/source/raysgd/v2/user_guide.rst

```diff
@@ -294,14 +540,41 @@ Here is an example:
 
     import torch
     import torchmetrics
-    import ray
+    from ray.util.sgd.v2 import Trainer
 
     def train_func(config):
         preds = torch.randn(10, 5).softmax(dim=-1)
         target = torch.randint(5, (10,))
+        # In a real use-case this would be passed into sgd.report()
+        # and processed with a Callback.
+        return torchmetrics.functional.accuracy(preds, target)
+
+    trainer = Trainer(backend="torch", num_workers=2)
+    trainer.start()
+    print(trainer.run(train_func))
+    # [tensor(0.3000), tensor(0.3000)]
+    trainer.shutdown()
+
+..
+    Advanced APIs
+    -------------
+
+    TODO
+
+    Training Run Iterator API
```

Comment:
On second thought, this might make more sense in the Monitoring Training/Logging section.

Reply:
Hmmm, I prefer to keep it here because there are other use-cases for this API e.g. integrations with other libraries. We can move it in the future after filling out the details and seeing if it fits better somewhere else then?

Reply:
I think we should link it from the logging section to here then.

Reply:
Done! (as a TODO since this section isn't written yet).